utils/package.json

```diff
@@ -21,6 +22,7 @@
     "pg-copy-streams": "^5.1.1",
     "pg-cursor": "^2.6.0",
     "pg-listen": "^1.7.0",
+    "stacks-encoding-native-js": "file:../../stacks-encoding-native-js",
```

Comment:
Shouldn't this point to your repo? I know it's temporary but just in case

Reply:
This debug code has since been converted into unit tests in the stacks-encoding-native projects, so I just removed them from this `utils/` dir.